sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1480,34 +1459,9 @@ def CreateTrigger(
             options = {}
 
         collection_id, path, trigger = self._GetContainerIdWithPathForTrigger(collection_link, trigger)
-        return self.Create(trigger, path, "triggers", collection_id, None, options, **kwargs)
-
-    def UpsertTrigger(
-        self,
-        collection_link: str,
-        trigger: Dict[str, Any],
-        options: Optional[Mapping[str, Any]] = None,
-        **kwargs: Any
-    ) -> Dict[str, Any]:
-        """Upserts a trigger in a collection.
-
-        :param str collection_link:
-            The link to the document collection.
-        :param dict trigger:
-        :param dict options:
-            The request options for the request.
-
-        :return:
-            The upserted Trigger.
-        :rtype:
-            dict
-
-        """
```

Comment:
@annatisch not sure if you meant this one by the other method, but this one very literally had no uses (which is why I removed it) - it also doesn't seem like we support this method in other SDKs (upsert trigger)